packages/helper-grammar-regex-collection/test.js

```diff
@@ -418,7 +436,7 @@ const fixtures = {
 function fixturesIterator(fixturesList, next) {
   fixturesList.forEach(statement => {
     const text = Array.isArray(statement) ? statement[0] : statement;
-    const expected = Array.isArray(statement) ? statement[1] : ['foo'];
+    const expected = Array.isArray(statement) ? statement[1] : null;
```

Comment:
Maybe I'm missing something here, but do we still need the ternary operator here? It should be always an array, right? 

```js
const [text, expected] = statement;
```